src/toil_scripts/lib/urls.py

```diff
@@ -53,14 +65,13 @@ def s3am_upload(fpath, s3_dir, num_cores=1, s3_key_path=None):
     :param int num_cores: Number of cores to use for up/download with S3AM
     :param str s3_key_path: (OPTIONAL) Path to 32-byte key to be used for SSE-C encryption
     """
-    if not s3_dir.startswith('s3://'):
-        raise ValueError('Format of s3_dir (s3://) is incorrect: {}'.format(s3_dir))
+    require(s3_dir.startswith('s3://'), 'Format of s3_dir (s3://) is incorrect: {}'.format(s3_dir))
```

Comment:
Not necessarily for this PR, but BPL's require() allows you to

```
require(s3_dir.startswith('s3://'), 'Format of s3_dir (s3://) is incorrect: %s',  s3_dir)
```

avoiding the string interpolation when requirement is satisfied.